src/ProcessManager/ChromeManager.php

```diff
@@ -30,13 +30,26 @@ final class ChromeManager implements BrowserManagerInterface
     private $arguments;
     private $options;
 
+    private $experimentalOptions = [];
+    private $extensions = [];
+
     public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null, array $options = [])
     {
         $this->options = array_merge($this->getDefaultOptions(), $options);
         $this->process = new Process([$chromeDriverBinary ?: $this->findChromeDriverBinary(), '--port='.$this->options['port']], null, null, null, null);
         $this->arguments = $arguments ?? $this->getDefaultArguments();
     }
 
+    public function setExtensions(array $extensions)
+    {
+        $this->extensions = $extensions;
```

Comment:
Is there a need to verify that the paths in `$extensions` reference files that can be read? What happens if an invalid or unreadable path is present?